02.calendar/calendar.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+
+const main = () => {
+  const today = new Date();
+  const argv = minimist(process.argv.slice(2));
+  const year = argv["y"] || today.getFullYear();
+  const month = argv["m"] || today.getMonth() + 1;
+  const firstDay = new Date(year, month - 1);
+  const lastDay = new Date(year, month, 0);
+  displayHeader(year, month);
+  displayBlankSpaces(firstDay);
+  displayDays(firstDay, lastDay);
+  displayFooter();
```

Comment:
footerと呼べるほどのものではないので、

```suggestion
  console.log("\n");
```

だけで十分かも

Reply:
修正します!